addons/vehiclelock/functions/fnc_moduleSync.sqf

```diff
@@ -28,7 +28,7 @@ if !(_activated) exitWith {WARNING("Vehicle Lock Sync Module - placed but not ac
     params ["_syncedObjects"];
 
     private _listOfVehicles = _syncedObjects select {
-        (_x isKindOf "Car") || {(_x isKindOf "Tank") || {_x isKindOf "Helicopter"}}
+        _x isKindOf "Car" || {_x isKindOf "Tank" || {_x isKindOf "Helicopter" || {_x isKindOf "Plane"}}}
```

Comment:
```suggestion
        private _vehicle = _x;
        -1 < ["Car", "Tank", "Helicopter", "Plane"] findIf {_vehicle isKindOf _x}
```
Also I think this array of classes should become macro because it's used at least in 2 places.